main.js

```diff
@@ -1240,6 +1246,7 @@ class Parcel extends utils.Adapter {
             const sendungsArray = data.sendungen.map((sendung) => {
                 const sendungsObject = { id: sendung.id, name: sendung.name, status: sendung.status, source: "AMZ" };
                 sendungsObject.inDelivery = this.inDeliveryCheck(sendungsObject);
+                sendungsObject.delivered = sendung.detailedState.progressTracker.lastTransitionPercentComplete == 100? true : false;
```

Comment:
You should check for the state otherwise the adapter will crash with property of null